joern-cli/frontends/gosrc2cpg/src/test/scala/io/joern/go2cpg/passes/ast/ArraysTests.scala

```diff
@@ -405,4 +405,99 @@ class ArraysTests extends GoCodeToCpgSuite {
     val List(identifierNode) = assignmentCallNode.astChildren.isIdentifier.l.l
     identifierNode.code shouldBe "c"
   }
+
+  "be correct when array access using variable having single index" in {
+    val cpg = code("""
+        |package main
+        |func main() {
+        | value := myArray[index]
```

Comment:
This is not a valid code. Make use of valid code to initialise the array and then access to array element.